modules/core/24-host/channel_keys.go

```diff
@@ -28,6 +29,16 @@ func ChannelCapabilityPath(portID, channelID string) string {
 	return fmt.Sprintf("%s/%s", KeyChannelCapabilityPrefix, channelPath(portID, channelID))
 }
 
+// ChannelUpgradeTimeoutPath defines the path set by the upgrade initiator to determine when the UPGRADETRY step should timeout
```

Comment:
Should this be `TRYUPGRADE` now? 

```suggestion
// ChannelUpgradeTimeoutPath defines the path set by the upgrade initiator to determine when the TRYUPGRADE step should timeout
```